pkg/security/secl/rules/ruleset.go

```diff
@@ -264,6 +278,115 @@ func (rs *RuleSet) AddRules(parsingContext *ast.ParsingContext, rules []*RuleDef
 	return result
 }
 
+func (rs *RuleSet) validatePolicyRules(policyRules []*RuleDefinition) *multierror.Error {
```

Comment:
This function doesn't validate only the policyRules, it generate also some internal data. We should maybe find another name

Reply:
How about `populateFieldsWithRuleActionsData`?